app/app/wyoming.ts

```diff
@@ -12,7 +17,7 @@ import {
 } from './proto/hassmic';
 import {CheyenneSocket} from './cheyenne';
 import {DeviceEventEmitter} from 'react-native';
-import {UUIDManager} from './util';
+import AsyncStorage from '@react-native-async-storage/async-storage';
```

Comment:
I think it makes more sense to make this a field in the settings proto rather than creating a new storage key directly.

If you haven't already (and I don't think I documented this anywhere yet, so that's my bad) - installing [pre-commit](https://pre-commit.com/) will automatically rebuild the generated proto files and do some other useful cleanup whenever you make a commit. You can also just run `./scripts/proto_regen.sh` after you change the `.proto` files to regenerate all of the client code.